Source/WebCore/platform/encryptedmedia/CDMInstanceSession.h

```diff
@@ -47,6 +47,14 @@ class SharedBuffer;
 
 enum class CDMKeyGroupingStrategy : bool;
 
+enum class CDMInstanceSessionLoadFailure : uint8_t {
```

Comment:
why move this out of the class, rather than use `[nested]` in the serialisation definition?

Reply:
This does need to be moved out because it is serialized outside of the scope of the parent class.  [Nested] makes it so that there is no declaration in the generated header, and we need the declaration in the generated header in this case.